app/pages/organisation/contextCreator.js

```diff
@@ -9,7 +9,7 @@ export const getContext = ({ organisation }) => ({
   address: organisation.address ? organisation.address.split(',') : [],
   agreementSigned: organisation.isCatalogueAgreementSigned,
   columnInfo: manifest.columnInfo ? manifest.columnInfo : [],
-  addUserButtonHref: `/organisations/${organisation.organisationId}/adduser`,
+  addUserButtonHref: 'adduser',
```

Comment:
bit confused by this - how will it route to the right path?

Reply:
because you're already on the /organisations/${organisation.organisationId}/ page when you click this link

Reply:
this is why I'm concerned about this whole relative url thing, it seems a bit weird

Reply:
OK let's trust our tests and let's get this in. If things don't work then our tests need updating.